recipe/meta.yaml

```
@@ -1,33 +1,34 @@
 {% set name = "svgdigitizer" %}
-{% set version = "0.12.0" %}
+{% set version = "0.12.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/svgdigitizer-{{ version }}.tar.gz
-  sha256: 540e8801ef09bc9c164f434af25b910e4c70470ae57288791960082cd8708aef
+  sha256: e62917bf6b1b5a6a2f7b1833e500ffb8f6c1cad31808c7068707ba9e7463adf6
 
 build:
   entry_points:
     - svgdigitizer=svgdigitizer.entrypoint:cli
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  number: 1
+  number: 0
 
 requirements:
   host:
     - pip
     - python {{ python_min }}
+    - setuptools
   run:
     - astropy >=5,<6
```

```suggestion
    - astropy-base >=5,<6
```

Any reason not to do this here as well?